.github/workflows/test_and_deploy.yml

```diff
@@ -0,0 +1,126 @@
+name: Test and Deploy
+
+on:
+  # Run this workflow every time a PR is opened or a new commit is pushed to the PR
+  pull_request:
+  # Run this workflow every time a PR is merged to main or a release tag is added
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+env:
+  MAIN_PYVER: 3.8
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup conda env
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda config --set always_yes yes --set changeps1 no
+          conda update -q conda
+          conda info -a
+          conda install -q conda-build
+          conda create -n numba-scipy -c numba python=${{ env.MAIN_PYVER }} numba>=0.45 'scipy>=0.16,<1.4' flake8 pytest
+          conda activate numba-scipy
+          pip install -e .
```

Comment:
Want to add `--no-deps` before `-e`?  Should catch errors in specs.

Reply:
good idea!